pom.xml

```diff
@@ -479,6 +479,9 @@
             <redirectTestOutputToFile>true</redirectTestOutputToFile>
             <trimStackTrace>false</trimStackTrace>
             <argLine>${jvm.options}</argLine>
+            <excludes>
```

Comment:
Why is this needed?

Reply:
I found that inner static test classes are not executed. Tests inside LoggerTest are ignored.

https://github.com/junit-team/junit5/issues/1377#issuecomment-381964988
https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#excludes

Reply:
From other pull request
https://app.circleci.com/pipelines/github/OpenFeign/feign/2890/workflows/95131078-58e3-4743-8fbf-b8c17d602ef5/jobs/8126?invite=true#step-105-18848_57

From this one
https://app.circleci.com/pipelines/github/OpenFeign/feign/2892/workflows/216b89f1-c576-4130-a322-cf2c23646607/jobs/8128?invite=true#step-105-21007_57